packages/wrangler/src/pages/dev.tsx

```diff
@@ -228,6 +230,12 @@ export const Handler = async ({
 		if (!existsSync(scriptPath)) {
 			logger.log("No functions. Shimming...");
 			scriptPath = resolve(__dirname, "../templates/pages-shim.ts");
+		} else {
+			workerJsBuild({
```

Comment:
This will only run once at start-up. It feels like it should be something that should happen as part of the `useEsbuild` hook. Possibly some flag we pass through? `disableImports`?

However, I'd be happy to accept this PR as-is right now, since this restriction may not exist forever anyway. I imagine if/when we come to add support for WASM in Pages Functions, this probably needs to be 'fixed' generally (but don't hold me to that).